tests/Utilities_test.cpp

```
@@ -255,6 +255,53 @@ SCENARIO("Printing Delaunay triangulations" * doctest::test_suite("utilities"))
+{
+  GIVEN("A bipartite cycle and two components with the same node degrees")
+  {
+    std::vector<std::string> const vertex_bases(4, "v:coincident:0");
+    std::vector<std::string> const cell_bases(4, "c:0");
+    std::vector<std::vector<std::size_t>> const cycle{
+        {0, 1},
+        {1, 2},
+        {2, 3},
+        {3, 0}
+    };
+    std::vector<std::vector<std::size_t>> const reordered_cycle{
```

Cppcheck (2.21.0)

[style] The function 'edges' is never used.

(unusedFunction)